### PR TITLE
Documentation:Expanded the docker compose instructions and details

### PR DESCRIPTION
**Pull request by kbventures**
Expanded the docker compose instructions and details

## Changes proposed

The steps and options to use the project locally were expanded for clarity

We added git as a requirement. 
We included docker desktop as an option because it's installable on all OS's.
We expanded the the details of the steps to make them more clear.
We included an optional drop down option which covers the steps needed to run the next.js app
locally and a mongoDB instance in docker  when Mongo Compass is needed as it won't work
with Docker compose.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

read1

read2

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8608"></a>

### CODE REVIEW

**Comment by JayPatil9** [comment]
Suggestion:-

```diff
-  <summary>Click me for instructions</summary>
+   <summary>Click here for instructions</summary>
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
It's hard to manage PRs without an issue linked. Please link the issue you're solving/assigned in the PR description. 

I recommend reading our [contributing guidelines](https://github.com/EddieHubCommunity/LinkFree/blob/main/CONTRIBUTING.md#issues--pull-requests-not-for-profiles) before creating PRs not for profiles. 

--
We tend to keep the README short, and already had [docker docs](https://linkfree.io/docs/environments/local-development#docker-compose) on LinkFree, So I don't think if these changes are required!

**Comment by eddiejaoude** [comment]
Sorry for the confusion, it is my fault, I mentioned to Ken on Discord that there was no need to raise an issue for these changes - I broke the rules 🤦‍♂️ I created an issue for it (@kbventures please can you comment on the issue and I can assign it to you) https://github.com/EddieHubCommunity/LinkFree/issues/8612

File: README.md [hunk 1]
```diff
@@ -81,12 +81,42 @@ This will allow you to run your favourite IDE but not have to install any depend
...
 #### Commands
 
-1. `docker compose up` 
+1. `git clone https://github.com/EddieHubCommunity/LinkFree`
+
+2. `cd LinkFree`
+
+3. `docker compose up` 
+
+4. In your browser on localhost:3000 you should now see the project up and running.
```
Comment (CBID2) [inline README.md:95]: [Proofreading] This is a good start but consider shortening the paragraph a bit to make the direction clearer. 
```suggestion
4. Once localhost:3000 shows up in your browser,  the project should be up and running.
```

File: README.md [hunk 2]
```diff
@@ -81,12 +81,42 @@ This will allow you to run your favourite IDE but not have to install any depend
...
+
+5. Now you need to upload the data in your mongoDB instance. `localhost:3000/api/system/reload?secret=development`
+
+6. Recheck localhost:3000 to confirm data is uploaded, you should see current amount of active users.
```
Comment (CBID2) [inline README.md:99]:
```suggestion
6. Check localhost:3000 again to confirm that the data is uploaded. You should see current amount of active users. 
```

File: README.md [hunk 3]
```diff
@@ -81,12 +81,42 @@ This will allow you to run your favourite IDE but not have to install any depend
...
+
+3. `docker compose up` 
+
+4. In your browser on localhost:3000 you should now see the project up and running.
+
+5. Now you need to upload the data in your mongoDB instance. `localhost:3000/api/system/reload?secret=development`
+
+6. Recheck localhost:3000 to confirm data is uploaded, you should see current amount of active users. 
+
+> **Warning**: [MongoDB Compass](https://www.mongodb.com/products/compass) will not not be available with the above setup. You'll need to run the next.js project that we just cloned locally and a mongoDB image inside a docker container in order to access
```
Comment (CBID2) [inline README.md:101]:
```suggestion
> **Warning**: [MongoDB Compass](https://www.mongodb.com/products/compass) will not be available with the above setup. You'll need to run the next.js project that we just cloned locally and a mongoDB image inside a docker container in order to  gain access
```

File: README.md [hunk 4]
```diff
@@ -81,12 +81,42 @@ This will allow you to run your favourite IDE but not have to install any depend
...
+  ### Installation
+  1. `npm install`
+  
+  2. Modify.env file `LINKFREE_MONGO_CONNECTION_STRING` value with `"mongodb://localhost:27017/linkfree"`
+  
+  3. `docker run --name my_mongodb_instance -p 27017:27017 -d mongo`
+  
+  4. `npm run dev`
+  
+  5. In the browser `localhost:3000/api/system/reload?secret=development`
```
Comment (CBID2) [inline README.md:116]: [Question] Is this supposed appear in the browser? Consider doing the following edit to clarify this direction 
```suggestion
  5. In the browser, you should see`localhost:3000/api/system/reload?secret=development`
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hey @kbventures! :) Welcome to EddieHub! :) I made some suggestions on how to revise some of the directions. Other than that, you're off to a great start! :)

**Comment by kumarsonsoff3** [comment]
> Sorry for the confusion, it is my fault, I mentioned to Ken on Discord that there was no need to raise an issue for these changes.

It's okay, Eddie. I've gone through the conversation in Discord for the context to the changes! 🙂

**Review by Shivamdwivedi1403 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (eddiejaoude) [reply README.md:95]: I think this is just a personal preference, so I will mark as resolved

File: README.md (hunk 2 above)
Comment (eddiejaoude) [reply README.md:99]: I think this is a personal preference also, so I will mark as resolved

File: README.md (hunk 3 above)
Comment (eddiejaoude) [reply README.md:101]: Good spot on the `not not` 👍

**Review by eddiejaoude — APPROVED** [review]
Looks great 👍

**Pull request merged**